features/363-glossary/glossary.md

```diff
@@ -0,0 +1,53 @@
+Glossary
+========
+
+In this document, we define a list of terms to be used in communication.
+
+Objective
+---------
+
+In various documents, we will use different terms to refer to concepts in Skygear:
+- Documentation
+- Technical discussions
+
+It would be clearer to use widely agreed terms to refer to specific concepts.
+
+Glossory
+--------
+- **Skygear Cluster**: the Skygear services (Gears, Gateway, and Controller), hosted on Kubernetes.
+- **Skygear Controller**: the management interface of Skygear Cluster.
+- **Skygear Portal**: the web management interface of Skygear Cluster.
+- **Skygear App**: a collection of microservices on Skygear Cluster, can omit 'Skygear' prefix if context if clear.
+- **Skygear Gateway**: the proxy which incoming traffic to Gears and Microservices must go through.
+- **Gear**: officially-supported multi-tenant-aware services.
+    - **Auth Gear**
+    - **Asset Gear**
+- **Deployment**: a unit of executable code belong to an app.
+    - **Microservices**
+- **Developer**: the developer of a Skygear App.
+- **User**: the user of a Skygear App.
+- **Secret**: confidential key value pair belonging to an app.
+- **Collaborator**: developer with management access to app.
+- **App Config**: configuration to gears.
```

Comment:
Currently we call the config **user config**, I opened an issue to update the config name to **app config**. Since this is another breaking change which has higher priority. https://github.com/SkygearIO/features/issues/402